models/DataObject/ClassDefinition/Data/RgbaColor.php

```diff
@@ -456,4 +456,18 @@ public function unmarshalAfterDecryption($value, $object = null, $params = [])
     {
         return Serialize::unserialize($value);
     }
+
+    /**
+     * @param Model\DataObject\Data\RgbaColor|null $oldValue
+     * @param Model\DataObject\Data\RgbaColor|null $newValue
+     *
+     * @return bool
+     */
+    public function isEqual($oldValue, $newValue)
+    {
+        $oldValue = $oldValue instanceof Model\DataObject\Data\RgbaColor ? $oldValue->getHex(true, true) : null;
```

Comment:
minor improvement, do it without hash (second parameter = false), there is no benefit of adding it